tests/Content/ContentControllerTest.php

```diff
@@ -0,0 +1,558 @@
+<?php
+
+namespace Railroad\MusoraApi\Tests\Content;
+
+use Carbon\Carbon;
+use Railroad\MusoraApi\Tests\TestCase;
+use Railroad\Railcontent\Factories\ContentContentFieldFactory;
+use Railroad\Railcontent\Factories\ContentFactory;
+use Railroad\Railcontent\Services\ContentService;
+
+class ContentControllerTest extends TestCase
+{
+    /**
+     * @var ContentFactory
+     */
+    protected $contentFactory;
+
+    /**
+     * @var ContentContentFieldFactory
+     */
+    protected $contentFieldFactory;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->contentFactory = $this->app->make(ContentFactory::class);
+        $this->contentFieldFactory = $this->app->make(ContentContentFieldFactory::class);
+    }
+
+    public function test_first()
+    {
+        $this->assertTrue(true);
+    }
+
+    public function test_pull_not_existing_content_endpoint()
+    {
+        $response =
+            $this->actingAs()
+                ->get(
+                    '/musora-api/content/' . rand()
+                );
+
+        $response->assertStatus(200);
```

Comment:
I think this should be a 404. If a related content does not exist, the content that linked to it should find its way to an error log to be reviewed and updated.